pkg/azurefile/controllerserver.go

```diff
@@ -20,10 +20,13 @@ import (
 	"context"
 	"fmt"
 	"net/url"
+	"os"
+	"os/exec"
 	"strconv"
 	"strings"
 	"time"
 
+	fileutil "sigs.k8s.io/azurefile-csi-driver/pkg/util"
```

Comment:
this is same as L30, just use `volumehelper `

Reply:
done